code/parse/parselo.cpp

```diff
@@ -2902,36 +2902,43 @@ size_t stuff_int_list(int *ilp, size_t max_ints, int lookup_type)
 					num = ship_name_lookup(str.c_str());	// returns index of Ship[] entry with name
 					if (num < 0)
 						error_display(1, "Unable to find ship %s in stuff_int_list!", str.c_str());
```

Comment:
This will still display an Error.  Is that what `SHIP_TYPE` did before?

Reply:
It is, yeah. It would've hard errored in that case before as well.

Reply:
Since I'm a fan of consistency, would there be any problem with making this a Warning to match the others?

Reply:
Fine by me.